python/seldon_core/metrics.py

```diff
@@ -14,15 +13,15 @@
 )
 from prometheus_client.utils import floatToGoString
 
-logger = logging.getLogger(__name__)
-
-NONIMPLEMENTED_MSG = "NOT_IMPLEMENTED"
+from seldon_core.env_utils import (
```

Comment:
we could just (if we put this into `utils)`
```
from seldon_core import utils
```

and then use `utils.get_image_name` wherever needed

Though both works and and usually over-import so does not require a change now.